docs/includes/topic_hierarchical-port-binding.rst

```diff
@@ -0,0 +1,136 @@
+
+- Administrator access to both BIG-IP device(s) and OpenStack cloud.
+
+- Software Defined Networking (SDN) service.
+
+- F5 :ref:`agent <agent:home>` and :ref:`service provider driver <install-f5-lbaasv2-driver>` installed on the Neutron controller and all other hosts for which you want to provision LBaaS services.
+
+- Knowledge of `OpenStack Networking <http://docs.openstack.org/liberty/networking-guide/>`_ concepts.
+
+- Knowledge of BIG-IP `system configuration`_, `local traffic management`_, & `device service clustering`_.
+
+
+Caveats
+-------
+
+- In release v |release| of the F5 LBaaSv2 driver and agent, ``VLAN`` is the only supported ML2 network type.
+
+- Each F5 agent managing a BIG-IP :term:`device service cluster` must have the same ``f5_network_segment_physical_network`` setting. [#]_
+
+-  If multiple F5 agents are managing the same environment, all of the agents must use the same binding settings (in other words, either the default global segmentation bindings or hierarchical port binding). [#]_
+
+
+Configuration
+-------------
+
+1. Edit the :ref:`Agent Configuration File`:
+
+.. code-block:: text
+
+    $ sudo emacs /etc/neutron/services/f5/f5-openstack-agent.ini
+
+
+2. Configure the heirarchical port binding settings in the :ref:`L2/L3 Segmentation Modes` section of the config file.
+
+.. table:: Hierarchical Port Binding settings
+
+    +--------------------------------------+-------------------------------------------+---------------+
+    | Setting                              | Description                               | Default Value |
+    +======================================+===========================================+===============+
+    |f5_network_segment_physical_network   || The name of the network segment in which | None          |
+    |                                      || the agent will manage BIG-IP(s).         |               |
+    +--------------------------------------+-------------------------------------------+---------------+
+    |f5_network_segment_polling_interval   || Seconds between polling Neutron for a    | 10            |
+    |                                      || ``network_id`` to ``segmentation_id``    |               |
+    |                                      || mapping.                                 |               |
+    |                                      || See :ref:`disconnected services`.        |               |
+    +--------------------------------------+-------------------------------------------+---------------+
+    |f5_network_segment_gross_timeout      || Maximum seconds to wait for a network to | 300           |
+    |                                      || be bound before the LBaaS request fails. |               |
+    |                                      || See :ref:`disconnected services`.        |               |
+    +--------------------------------------+-------------------------------------------+---------------+
+
+
+
+.. topic:: Example
```

Comment:
Suggest the example show a customer how to configure settings if they want to run in "disconnected service" mode.  Uncomment the 3 settings, and supply sample text.